snooty.toml

```diff
@@ -23,3 +26,11 @@ driver = "node"
 driver-long = "MongoDB Node.js driver"
 driver-short = "Node.js driver"
 mdb-server = "MongoDB Server"
+current-url-base = "https://www.mongodb.com/docs/drivers/node/current"
+
+[[banners]]
+targets = ["*"]
+variant = "info"
+value = """\
+    This version of the documentation is archived and no longer supported. View the `current documentation <{+current-url-base+}>`__ to learn how to `upgrade your version of the {+driver-long+} <{+current-url-base+}/upgrade/>`__.\
+    """
```

Comment:
I think if they're upgrading from one of these older versions it's important for them to see the breaking changes first. Especially since we have the "Before you upgrade..." part in the top section.

That being said I do agree that it's a little confusing that a page called Upgrade Driver Versions doesn't actually tell how to perform the upgrade. Maybe we should add a section on that page stating (or linking to somewhere that states) how to actually install the new version.

Reply:
Could you create a ticket to add this section to the upgrade page?

Reply:
Created:
https://jira.mongodb.org/browse/DOCSP-31571